pilot/pkg/networking/core/v1alpha3/cluster_builder_test.go

```diff
@@ -113,6 +113,37 @@ func TestApplyDestinationRule(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:        "destination rule with pass through subsets",
+			cluster:     &cluster.Cluster{Name: "foo", ClusterDiscoveryType: &cluster.Cluster_Type{Type: cluster.Cluster_STATIC}},
+			clusterMode: DefaultClusterMode,
+			service:     service,
+			port:        servicePort[0],
+			networkView: map[string]bool{},
+			destRule: &networking.DestinationRule{
+				Host: "foo.default.svc.cluster.local",
+				Subsets: []*networking.Subset{
+					{
+						Name:   "foobar",
+						Labels: map[string]string{"foo": "bar"},
+						TrafficPolicy: &networking.TrafficPolicy{
+							LoadBalancer: &networking.LoadBalancerSettings{
+								LbPolicy: &networking.LoadBalancerSettings_Simple{Simple: networking.LoadBalancerSettings_PASSTHROUGH},
+							},
+						},
+					},
+				},
+			},
+			expectedSubsetClusters: []*cluster.Cluster{
+				{
+					Name:                 "outbound|8080|foobar|foo.default.svc.cluster.local",
+					ClusterDiscoveryType: &cluster.Cluster_Type{Type: cluster.Cluster_ORIGINAL_DST},
+					EdsClusterConfig: &cluster.Cluster_EdsClusterConfig{
```

Comment:
why is there EDS config at all? is that expected? I would expect it works without it (since PassthroughCluster doesn't have it) - does envoy behave the same with/without?

Reply:
It is not needed. Copy paste. Will remove.